core/domain/suggestion_services_test.py

```diff
@@ -48,9 +52,11 @@
     from mypy_imports import suggestion_models
     from mypy_imports import user_models
 
-(suggestion_models, feedback_models, user_models) = (
+(suggestion_models, feedback_models, opportunity_models, user_models) = (
     models.Registry.import_models(
-        [models.NAMES.suggestion, models.NAMES.feedback, models.NAMES.user]
+        [models.NAMES.suggestion, models.NAMES.feedback,
+        models.NAMES.opportunity,
+        models.NAMES.user]
```

Comment:
Put the square brackets next to the normal brackets, so it will be:
```
([


])

Reply:
Done